### PR TITLE
[NUI] Modify NUI Seamless animation bug

### PR DESCRIPTION
**Pull request by huiyueun**
- Modify seamless animation using shared object.
- Add Seamless XAML sample(test/Tizen.NUI.Seamless.XAML)

Signed-off-by: huiyu.eun <huiyu.eun@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/internal/FrameBroker/DefaultFrameBroker.cs [hunk 1]
```diff
@@ -63,20 +69,65 @@ protected override void OnFrameResumed(FrameData frame)
...
+        {
+            base.OnFramePaused();
+            animation?.Stop();
+
+            ResetImage();
+
+            isAnimating = false;
+        }
+
+        private bool Timer_Tick(object source, Timer.TickEventArgs e)
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/FrameBroker/DefaultFrameBroker.cs:82]: from next time, please do not use "_"
please use pascal case.

File: src/Tizen.NUI/src/public/TransitionAnimations/TransitionAnimations.cs [hunk 2]
```diff
@@ -3,60 +3,139 @@
...
+        /// property of animation
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Property { get; set; }
+
+        /// <summary>
+        /// destination value of animation
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string DestValue { get; set; }
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/TransitionAnimations/TransitionAnimations.cs:38]: from next time, please do not use short word.
"DestValue" should be "DestinationValue"

File: src/Tizen.NUI/src/public/TransitionAnimations/TransitionAnimations.cs [hunk 3]
```diff
@@ -3,60 +3,139 @@
...
+        /// <summary>
+        /// Default style of animate image view.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<TransitionAnimationData> AnimationDataList
         {
-            return new Size(0, 0);
+            get
+            {
+                return animationDataList;
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/TransitionAnimations/TransitionAnimations.cs:78]: very good!
only open getter for collection type

File: src/Tizen.NUI/src/public/TransitionOptions/TransitionOptions.cs [hunk 4]
```diff
@@ -45,6 +48,23 @@ public TransitionOptions(Window window)
             mainWindow = window;
         }
 
+
+        /// <summary>
+        /// Set animated view of seamless animation.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public View MainAnimatedView
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/TransitionOptions/TransitionOptions.cs:56]: View and return type View is duplicated.
from next time, please use other naming. 
such as MainAnimatedTarget, MainAnimatedObject ?

**Review by dongsug-song — APPROVED** [review]

File: src/Tizen.NUI/src/internal/FrameBroker/DefaultFrameBroker.cs (hunk 1 above)
Comment (huiyueun) [reply src/Tizen.NUI/src/internal/FrameBroker/DefaultFrameBroker.cs]: unused function. I've removed it.

File: src/Tizen.NUI/src/public/TransitionAnimations/TransitionAnimations.cs (hunk 2 above)
Comment (huiyueun) [reply src/Tizen.NUI/src/public/TransitionAnimations/TransitionAnimations.cs]: Done.

File: src/Tizen.NUI/src/public/TransitionOptions/TransitionOptions.cs (hunk 4 above)
Comment (huiyueun) [reply src/Tizen.NUI/src/public/TransitionOptions/TransitionOptions.cs]: Done -> Change AnimatedTarget

**Pull request merged**